docs/source/advanced/query-batching.mdx

```diff
@@ -1,50 +1,49 @@
 ---
-title: Query batching
+title: Batching operations
```

Comment:
I'm always hesitating on these ones. Same for "Automatic Persisted Queries" vs "Automatic Persisted Operations". Using "Operations" is more correct but there is so much muscle memory around "Query" that I'm always afraid using "Operation" might confuse more than it helps.

Reply:
I hear you! I think this is a fair place to attempt to expand out to "operations" because the reader will have already familiarized themselves with queries. And I think it behooves us in the long run to get folks more comfortable with "operations" so we can _slightly_ disambiguate the term "query"